### PR TITLE
[DOCS] Modifying getting started tutorial content to work with 0.13.8+

### LINKED ISSUE #2358: Getting started example failing with config version 3.0
**Describe the bug**
Hi,
I made a fresh install of latest ge version (0.13.8) and followed the [getting started example](https://docs.greatexpectations.io/en/latest/guides/tutorials/getting_started/create_your_first_expectations.html). After I run the `scaffold` command and run the jupyter notebook, the last cell "Save & review the scaffolded Expectation Suite" throws an error at `results = context.run_validation_operator("action_list_operator", assets_to_validate=[batch])`

**Error message**:  
`DataContextError: No validation operator `action_list_operator` was found in your project. Please verify this in your great_expectations.yml`

**To Reproduce**
1. Clone the tutorial `git clone https://github.com/superconductive/ge_tutorials` and start postgres container
2. Initialize a new expectations project `great_expectations init` following the steps at the [tutorial docs](https://docs.greatexpectations.io/en/latest/guides/tutorials/getting_started/initialize_a_data_context.html)
3. Run the `scaffold` command `great_expectations suite scaffold taxi.demo` and execute notebook cells
4. See error at last cell

**Environment (please complete the following information):**
 - Operating System: Windows, Python 3.8.0
 - Great Expectations Version: [0.13.8]

**Additional context**
If I manually add the `action_list_operator` in `great_expectations.yml` as shown at the [Github example](https://github.com/superconductive/ge_tutorials/blob/ad93033d0ecf99d35834dafccfacf32bc9adfb9e/ge_getting_started_tutorial/great_expectations/great_expectations.yml#L40) the notebook cell is executed but then when I run a new checkpoint it shows a deprecation message: 
`You appear to be using a legacy capability with the latest config version (3.0).
    Your data context with this configuration version uses validation_operators, which are being deprecated.  Please update your configuration to be compatible with the version number 3.`

I couldn't find any similar issue. Thank you!

### PR DESCRIPTION
**Pull request by spbail**
- updating the tutorial content to work with class-based checkpoints
- Also switching the tutorial to using a CSV file instead of Docker while I'm at it, that's been in our backlog for a while.
- some additional minor tweaks and cleanup (removed some duplicated content etc)

This is also blocked on making legacy checkpoints work so we can continue to create and run them with legacy datasources, but after running pip install with 0.13.8+.

Matching tutorials repo update: https://github.com/superconductive/ge_tutorials/pull/14

### CODE REVIEW

**Comment by spbail** [comment]
This depends on Rob's checkpoint fix, but should be good to go once that's merged.

**Review by jcampbell — APPROVED** [review]

**Pull request merged**
